### PR TITLE
[Blog] Moving to Trunk

### PR DESCRIPTION
**Pull request by AutomatedTester**
Blog post describing the removal of the `master` branch

### Description
 
Moving the default development branch from `master` to `trunk`

### Motivation and Context

The Selenium community wants to make sure that we can improve what we do to make sure that we are inclusive.

### Types of changes

- [x] Blog Post
- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: site_source_files/content/blog/2020/moving-to-trunk-development.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
++++
+Description = "Why we are getting rid of `master` branch in favour of `trunk`"
+Title = "Moving to trunk"
```
Comment (shs96c) [inline site_source_files/content/blog/2020/moving-to-trunk-development.md:3]: Nit: Use "title case". `trunk` should be capitalised.

File: site_source_files/content/blog/2020/moving-to-trunk-development.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+AuthorLink = "https://twitter.com/AutomatedTester"
+tags = ["selenium"]
+categories = ["general"]
++++
+
+Since the project started we have been following [trunk based development](https://trunkbaseddevelopment.com/). This was a very natural fit when we were using SVN over a decade ago on Google Code.
+
+As Google Code shut down we moved to GitHub and the git model of doing things. We moved there mostly due to the gravity that GitHub had created in Open Source projects.
+
+Unfortunately, it meant that we followed the standard use of `master` as our trunk to work off. Now that Github, and services that use Github, have improved support for non-master branches as default we are moving our default branch to `trunk`. It describes how we, as a project, work and is a more inclusive term.
```
Comment (shs96c) [inline site_source_files/content/blog/2020/moving-to-trunk-development.md:15]: `s/Unfortunately,/This/` At the time it wasn't considered to be unfortunate, it's only with hindsight that this is true.

File: site_source_files/content/blog/2020/moving-to-trunk-development.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+We, as a project, want to our community to be inclusive and this is just one step in making sure we are.
```
Comment (shs96c) [inline site_source_files/content/blog/2020/moving-to-trunk-development.md:17]: "want to our community"? Is there a missing "make" in there?

Comment (shs96c) [reply site_source_files/content/blog/2020/moving-to-trunk-development.md:17]: Natural follow-up question: what other steps are being taken?

File: site_source_files/content/blog/2020/moving-to-trunk-development.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+AuthorLink = "https://twitter.com/AutomatedTester"
+tags = ["selenium"]
+categories = ["general"]
++++
+
+Since the project started we have been following [trunk based development](https://trunkbaseddevelopment.com/). This was a very natural fit when we were using SVN over a decade ago on Google Code.
+
+As Google Code shut down we moved to GitHub and the git model of doing things. We moved there mostly due to the gravity that GitHub had created in Open Source projects.
+
+This meant that we followed the standard use of `master` as our trunk to work off. Now that Github, and services that use Github, have improved support for non-master branches as default we are moving our default branch to `trunk`. It describes how we, as a project, work and is a more inclusive term.
```
Comment (diemol) [inline site_source_files/content/blog/2020/moving-to-trunk-development.md:15]:
```suggestion
This meant that we followed the standard use of `master` as our trunk to work off. Now that GitHub, and services that use GitHub, have improved support for non-master branches as default we are moving our default branch to `trunk`. It describes how we, as a project, work and is a more inclusive term.
```

**Pull request merged**